taxonomies/ingredients.txt

```diff
@@ -9565,7 +9565,7 @@ fr:lait maigre
 # description:en:Skimmed milk (British English) is made when all the cream is removed from whole milk. It tends to contain around 0.1% fat. Láit écrémé has a maximum of 0.5% fat.
 
 <en:milk
-en:skimmed milk
+en:skimmed milk, non-fat milk, nonfat milk
```

Comment:
I rather keep skimmed milk and non-fat milk separate, as they have different fat contents in the US market.

Reply:
Discussed on Slack: https://openfoodfacts.slack.com/archives/C06A7LENM/p1567333003041000
Merged skim milk and skimmed milk entries.